packages/react/src/blocks/Image.tsx

```diff
@@ -99,65 +99,64 @@ class ImageComponent extends React.Component<any> {
         {value => {
           const amp = value.ampMode
           const Tag: 'img' = amp ? ('amp-img' as any) : 'img'
+
+          const imageContents = (!lazy || this.state.load) && (
+            <Tag
+              {...(amp
+                ? ({
+                    layout: 'responsive',
+                    height:
+                      this.props.height ||
+                      (aspectRatio ? Math.round(aspectRatio * 1000) : undefined),
+                    width:
+                      this.props.width ||
+                      (aspectRatio ? Math.round(1000 / aspectRatio) : undefined)
+                  } as any)
+                : null)}
+              alt={this.props.altText}
+              key={
+                Builder.isEditing
+                  ? (typeof this.props.image === 'string' &&
+                      this.props.image.split('?')[0]) ||
+                    undefined
+                  : undefined
+              }
+              role={!this.props.altText ? 'presentation' : undefined}
+              css={{
+                objectFit: this.props.backgroundSize,
+                objectPosition: this.props.backgroundPosition,
+                ...(aspectRatio && {
+                  position: 'absolute',
+                  height: '100%',
+                  width: '100%',
+                  left: 0,
+                  top: 0
+                }),
+                ...(amp && {
+                  ['& img']: {
+                    objectFit: this.props.backgroundSize,
+                    ObjectPosition: this.props.backgroundPosition
+                  }
+                })
+              }}
+              className="builder-image"
+              src={this.props.image}
+              // TODO: memoize on image on client
```

Comment:
@steve8708 you want this comment removed as well?

Reply:
this one we can keep actually, would be good to add at some point but not urgent